### PR TITLE
Sprint7 update documentation in various places on the site

### PR DESCRIPTION
**Pull request by colbycocking**
# Before
- Users could not see game code
- It was unclear how to start the challenges
- Documentation that showed up first was repetitive and unnecessary 
- No description of what the CTF application is.

# Work Done
- [x] #197 
- [x] #198
- [x] #194
- [x] #200

### CODE REVIEW

File: ui/src/pages/error-pages/preLoginPage.jsx [hunk 1]
```diff
@@ -1,23 +1,31 @@
...
-            <p>Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical
-                Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at
-                Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a
-                Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the
-                undoubtable source.</p>
+            <h2>Welcome to Capture the Flag!</h2>
+
+            <p>Capture the flag is a web-based application that allows you to start learning the basics of
+              cyber
+              literacy. Here we have a collection of activities that you can interact with, whether its a
```
Comment (lgorence) [inline ui/src/pages/error-pages/preLoginPage.jsx:19]: Second to last word on the line, "its" should be "it's".

File: ui/src/pages/error-pages/preLoginPage.jsx [hunk 2]
```diff
@@ -1,23 +1,31 @@
...
+              familiar game
+              written by one of hour high school contributors or a security challenge written by some of
```
Comment (lgorence) [inline ui/src/pages/error-pages/preLoginPage.jsx:21]: "one of hour" -> "one of our"

**Review by lgorence — APPROVED** [review]
Looks good to me, just a couple minor spelling/grammar things to fix up prior to merge.

File: ui/src/pages/error-pages/preLoginPage.jsx (hunk 1 above)
Comment (colbycocking) [reply ui/src/pages/error-pages/preLoginPage.jsx]: Done

File: ui/src/pages/error-pages/preLoginPage.jsx (hunk 2 above)
Comment (colbycocking) [reply ui/src/pages/error-pages/preLoginPage.jsx]: I can't believe I did this lol

**Pull request merged**